src/shared/ui/Chip/index.styles.ts

```
@@ -42,4 +42,5 @@ export const Container = styled.div<StyledChipContainerProps>`
   ${({ $size }) => chipSizes[$size]};
   ${({ $variant }) => chipVariants[$variant]};
   white-space: nowrap;
+  cursor: ${({ onClick }) => onClick ? 'pointer' : 'default'};
```

ESLint

[error] Replace `onClick·?·'pointer'·:·'default'` with `(onClick·?·'pointer'·:·'default')`

(prettier/prettier)